db/migrate/20160419192134_devise_create_users.rb

```diff
@@ -0,0 +1,42 @@
+class DeviseCreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
```

Comment:
After I've done some research, it's OK to have a separate model for user information. We just have to be careful of not getting N+1 queries when displaying info about user.